src/data/markdown/docs/30 xk6-browser/01 xk6-browser/01 Browser.md

```diff
@@ -3,22 +3,17 @@ title: "Browser"
-| [browser.version()](/javascript-api/xk6-browser/browser/version)                               | Returns the browser application's version.                                                                      |
+| Method                                                                                                                                                                         | Description                                                                                                                                           |
+|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------|
+| <nobr>[browser.close()](/javascript-api/xk6-browser/browser/close)</nobr>                                                                                                      | Closes the browser and all of its pages (if any were opened).                                                                                         |
+| <nobr>[browser.contexts()](/javascript-api/xk6-browser/browser/contexts)</nobr>                                                                                                | Allows you to access all open [BrowserContexts](/javascript-api/xk6-browser/browsercontext/)s.                                                        |
+| <nobr>[browser.isConnected](/javascript-api/xk6-browser/browser/isconnected)</nobr>                                                                                            | Indicates whether the [CDP](https://chromedevtools.github.io/devtools-protocol/) connection to the browser process is active or not.                  |
+| <nobr><span title="Work in progress">🚧</span> [browser.newContext([options])](/javascript-api/xk6-browser/browser/newcontext/)</nobr>                                         | Creates and returns a new [BrowserContexts](/javascript-api/xk6-browser/browsercontext/).                                                             |
+| <nobr><span title="Work in progress">🚧</span> [browser.newPage([options])](/javascript-api/xk6-browser/browser/newpage)</nobr>                                                | Creates a new [Page](/javascript-api/xk6-browser/page/) in a new [BrowserContexts](/javascript-api/xk6-browser/browsercontext/) and returns the page. |
+| <nobr><span title="Work in progress">[🚧](https://github.com/grafana/xk6-browser/issues/96)</span> [browser.on('disconnected')](/javascript-api/xk6-browser/browser/on)</nobr> | Detects events from the browser application.                                                                                                          |
```

Comment:
This one is actually done, see https://github.com/grafana/xk6-browser/pull/268. The link you added is for `BrowserContext.on()` which is still pending, but that's for the `BrowserContext` page, not here.

Reply:
Resolved in: 9a2dc280ae6320fe8ea2b9265abeb57bffd90b84